incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -262,10 +263,27 @@ func (r *HierarchyReconciler) syncLabel(log logr.Logger, nsInst *corev1.Namespac
 		return
 	}
 
+	// pre-define HNC label suffix
```

Comment:
Nit: I'd call the the "depth label suffix," not the "HNC label suffix," since HNC has lots of suffixes.

Reply:
updated

Reply:
sorry, the previous variable name is "labelDepthSuffix". Currently i change according to it. Not sure about the naming here, should i change to "depthLabelSuffix" ?

Reply:
I just just talking about the comment, the code lgtm. I left a more precise comment in my later review.